### DISCUSSION TITLE
[공유] 304 Not Modified와 관련된 HTTP 헤더

### DISCUSSION THREAD

**Discussion by nbalance97** [woowacourse-study/2022-http-web-basic-for-all-developer#32] [General]
클라이언트의 헤더에 If-Modified-Since가 있으면, 서버가 지정된 날짜 이후에 수정된 경우에는 200 상태 코드와 요청 리소스를 돌려주고, 수정되지 않았다면 304 응답을 한다고 합니다😀

https://developer.mozilla.org/ko/docs/Web/HTTP/Headers/If-Modified-Since

**Comment by kth990303** [comment]
304 Not Modified인 이유가 있었네요 :) 좋은 글 감사합니다.
